src/systemctl/systemctl-enable.c

```diff
@@ -208,7 +210,10 @@ int verb_enable(int argc, char *argv[], void *userdata) {
                 }
 
                 if (send_runtime) {
-                        r = sd_bus_message_append(m, "b", arg_runtime);
+                        if (streq(method, "DisableUnitFilesWithFlagsAndInstallInfo"))
```

Comment:
Is the conditionalization necessary?
With the commit, now only `DisableUnitFilesWithFlagsAndInstallInfo` is used, right?

Reply:
I don't really understand what you mean 🤔

There are other verbs that have `send_runtime` set to true, and they don't use a `WithFlags` variant of method. So they still need a boolean rather than flags as argument, per my understanding?

Reply:
ah, sorry for the noise. that's completely my misunderstanding.